pkg/trace/test/testutil/trace.go

```diff
@@ -115,3 +126,46 @@ func GetTestTraces(traceN, size int, realisticIDs bool) pb.Traces {
 	}
 	return traces
 }
+
+// GetTestTraceChunks returns a []TraceChunk that is composed by ``traceN`` number
+// of traces, each one composed by ``size`` number of spans.
+func GetTestTraceChunks(traceN, size int, realisticIDs bool) []*pb.TraceChunk {
+	traces := GetTestTraces(traceN, size, realisticIDs)
+	traceChunks := make([]*pb.TraceChunk, 0, len(traces))
+	for _, trace := range traces {
+		traceChunks = append(traceChunks, &pb.TraceChunk{
+			Spans: trace,
+		})
+	}
+	return traceChunks
+}
+
+// GetTraceChunkWithSpan wraps a `span` with pb.TraceChunk
+func GetTraceChunkWithSpan(span *pb.Span) *pb.TraceChunk {
```

Comment:
Can we drop the `Get*` prefixes please? (https://golang.org/doc/effective_go#Getters)